### ISSUE TITLE
How to evaluate results on test set although we don't have ground truths for test data?

### ISSUE THREAD

**Issue by farooqrasheed123** [pjl1995/CTracker#14]
How can we evaluate MOT16/MOT17 test data as we don't have ground truth for that? During training, we have ground truths and we can evaluate results on training data but how to evaluate on test data of MOT16 or MOT17 without ground truths ?

**Comment by pjl1995** [comment]
You can register an account on the website https://motchallenge.net/ and submit the results of test data for evaluation.

**Comment by farooqrasheed123** [comment]
Thanks a lot.
